### ISSUE TITLE
NVDA announces Combo box before reading adres bar in recent versions of Nightly

### ISSUE THREAD

**Issue by lukaszgo1** [nvaccess/nvda#8909]
### Steps to reproduce:
1. Open Nightly 65.
2. Press CTRL+L to move to the adress bar,
### Actual behavior:
NVDA announces Combo box before reading 'search with google or enter adres'
### Expected behavior:
The combo box shouldn't be announced.
#### NVDA Installed/portable/running from source:
Installed
#### NVDA version:
Version: alpha-16183,c3ce6a38
#### Windows version:
Windows 7 x64
#### Name and version of other software in use when reproducing the issue:
Nightly 65.0a1 (2018-11-01) (64-bit)
#### Other information about your system:
At first I've thought, that it is something which should be reported on the Mozilla Bug Tracker, but after testing with JAWS 2018 this problem is specific to NVDA.
#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA?

No

**Comment by lukaszgo1** [comment]
cc @jcsteh

**Comment by MarcoZehe** [comment]
I actually think the way it is now is more appropriate, since the name "Enter address or search with Google" directly relates to what should be typed.

**Comment by lukaszgo1** [comment]
I believe that I am simply used to having the name reported first, and then the control type, but it is just a personal preference. The only argument for reverting this change which come to my mint is the rule to present changeable information first, and because it is and always would be editable combo box it might make sense to have this spoken at the end.

**Comment by MarcoZehe** [comment]
The separate search box might go away soon, in fact it is not enabled by default in new profiles. The updates to the URL bar were most likely not applied to this legacy search bar.

**Comment by josephsl** [comment]
Hi,

Six years later...

As this concerns a specific version of Firefox (you should only use Firefox Nightly if you know what you're doing), it would not hold true in 2024. Closing.

**Comment by Adriani90** [comment]
This is reproducible in Firefox 128 with NVDA 2024.3 Beta, reopening.

**Comment by josephsl** [comment]
Hi, CC @jcsteh

**Comment by jcsteh** [comment]
After [Mozilla bug 1938631](https://bugzilla.mozilla.org/show_bug.cgi?id=1938631) (Firefox 150), the address bar is no longer a textbox inside of a combobox (ARIA 1.1 combobox), but is instead a single combobox with the editable state (ARIA 1.2 combobox). That simplifies what NVDA reports here.
